test/k8s-e2e/descriptions/blacklist.desc

```diff
@@ -0,0 +1,57 @@
+[sig-scalability] Density [Feature:ManualPerformance] should allow starting 30 pods per node using ReplicationController with 0 secrets, 0 configmaps, 0 token projections, and 2 daemons
+[sig-scalability] Density [Feature:ManualPerformance] should allow starting 50 pods per node using ReplicationController with 0 secrets, 0 configmaps, 0 token projections, and 0 daemons
+[sig-scalability] Density [Feature:Performance] should allow starting 30 pods per node using ReplicationController with 0 secrets, 0 configmaps, 0 token projections, and 0 daemons
+[sig-scalability] Load capacity [Feature:ManualPerformance] should be able to handle 3 pods per node ReplicationController with 0 secrets, 0 configmaps and 0 daemons
+[sig-scalability] Load capacity [Feature:ManualPerformance] should be able to handle 30 pods per node Deployment.extensions with 0 secrets, 0 configmaps and 0 daemons
+[sig-scalability] Load capacity [Feature:ManualPerformance] should be able to handle 30 pods per node Deployment.extensions with 0 secrets, 2 configmaps and 0 daemons
+[sig-scalability] Load capacity [Feature:ManualPerformance] should be able to handle 30 pods per node Deployment.extensions with 2 secrets, 0 configmaps and 0 daemons
+[sig-scalability] Load capacity [Feature:ManualPerformance] should be able to handle 30 pods per node Job.batch with 0 secrets, 0 configmaps and 0 daemons
+[sig-scalability] Load capacity [Feature:ManualPerformance] should be able to handle 30 pods per node Random with 0 secrets, 0 configmaps and 0 daemons
+[sig-scalability] Load capacity [Feature:ManualPerformance] should be able to handle 30 pods per node Random with 0 secrets, 0 configmaps and 0 daemons with quotas
+[sig-scalability] Load capacity [Feature:ManualPerformance] should be able to handle 30 pods per node ReplicationController with 0 secrets, 0 configmaps and 0 daemons with quotas
+[sig-scalability] Load capacity [Feature:ManualPerformance] should be able to handle 30 pods per node ReplicationController with 0 secrets, 0 configmaps and 2 daemons
+[sig-scalability] Load capacity [Feature:Performance] should be able to handle 30 pods per node ReplicationController with 0 secrets, 0 configmaps and 0 daemons
+[sig-scheduling] SchedulerPredicates [Serial] validates local ephemeral storage resource limits of pods that are allowed to run [Feature:LocalStorageCapacityIsolation]
+[sig-network] NoSNAT [Feature:NoSNAT] [Slow] Should be able to send traffic between Pods without SNAT
+[sig-storage] In-tree Volumes [Driver: iscsi][Feature:Volumes] [Testpattern: Inline-volume (default fs)] subPath should be able to unmount after the subpath directory is deleted
+[sig-scheduling] ResourceQuota [Feature:Initializers] should create a ResourceQuota and capture the life of an uninitialized pod.
+[sig-network] [Feature:PerformanceDNS][Serial] Should answer DNS query for maximum number of services per cluster
```

Comment:
This is only blacklisted until the etccd problems are fixed?

Reply:
Yes, depending on decision of etcd and PO. Eventually we say, that we cannot support this for current time being.